web/cypress/integration/route/create-edit-delete-route.spec.js

```diff
@@ -80,11 +81,28 @@ context('Create and Delete Route', () => {
     cy.url().should('contains', 'routes/list');
   });
 
+  it('edit the route', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+    cy.contains('Edit').click();
+    // input new name and newdescription
+    cy.get('#name').clear().type(newname);
+    cy.get('#desc').clear().type('new desc');
+    cy.contains('Next').click();
+    cy.contains('Next').click();
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.contains('SubmitSuccessfully');
```

Comment:
no space after “submit”？

Reply:
> no space after “submit”？

refer to cypress‘ documents: ` It’s automatically waiting and retrying because it expects the content to eventually be found in the DOM. It doesn’t immediately fail!`
https://docs.cypress.io/guides/getting-started/writing-your-first-test.html#Step-2-Query-for-an-element

Reply:
- [ ] Update this text with space;

Reply:
fixed